openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/doc.go

```diff
@@ -0,0 +1,13 @@
+package managementcpusoverride
+
+// The ManagementCPUOverride admission plugin replaces pods containers CPUs requests with a new management resource
+// under the pod with the workload annotation under allowed namespace.
```

Comment:
It applies to all pods that
1. are in an allowed namespace
2. and have the workload annotation.

Reply:
done